tools/dump_tool/dump_tool.c

```diff
@@ -389,30 +454,97 @@ int main(int argc, char *argv[])
 		goto tty_lock_err;
 	}
 
-	if (do_handshake(dev_fd) != 0) {
-		printf("Target Handshake Failed\n");
-		goto handshake_err;
-	}
+	printf("Target device locked and ready to DUMP!!!\n");
+
+	while (1) {
+		printf("Chose from the following options:-\n1. RAM Dump\n2. Userfs Dump\n3. Both Dumps\n4. Exit Dump Tool\n5. Reboot TARGET Device\n");
+		fflush(stdout);
+		scanf("%d", &choice);
+		getchar();
+
+		switch (choice) {
+		case 2:
```

Comment:
why is it not maintained in sequence ? any reason

Reply:
Trying to club cases which have similar statements for execution.